lisa/tools/hyperv.py

```diff
@@ -170,6 +176,29 @@ def create_vm(
         if not is_ready:
             raise LisaException(f"VM {name} did not start")
 
+    def stop_vm(self, name: str) -> None:
+        # stop vm
+        self._run_hyperv_cmdlet("Stop-VM", f"-Name {name} -Force", force_run=True)
+
+    def restart_vm(
+        self,
+        name: str,
+    ) -> None:
+        # restart vm
+        self._run_hyperv_cmdlet("Restart-VM", f"-Name {name} -Force", force_run=True)
+
+    def enable_device_passthrough(self, name: str, mmio_mb: int = 5120) -> None:
```

Comment:
1. When is it need to be enabled? Does it need to disable?
2. the default value of mmio_mb, is it recommended? If it uses the physical memory, it maybe too many to impact VM mem usage.

Reply:
@squirrelsc 

`enable_device_passthrough` is to set config so VM can DDA supportable. We will execute this when there is a device pool configured from runbook. When Node will be deleted, all the devices would be removed from it and assigned back to host by platform.

MMIO space is virtual one can be assigned to any value. I have seen mostly in MBs but we are setting it as default 5GB as of now. It won't reserve anything on physical machine.